From f71fa6e9159b4514c9a1bec813ac166f47d58ed9 Mon Sep 17 00:00:00 2001
From: Alexander Tsoy <alexander@tsoy.me>
Date: Mon, 6 Mar 2023 00:57:20 +0300
Subject: [PATCH] configure: drop AC_FUNC_STAT macro

This macro is obsolescent and doesn't work with
-Werror=implicit-function-declaration compiler flag
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2fd443a..f891d81 100644
--- a/configure.ac
+++ b/configure.ac
@@ -496,7 +496,6 @@ if test $inaddrt = no ; then
 fi
 # Checks for library functions.
 AC_FUNC_MEMCMP
-AC_FUNC_STAT
 AC_FUNC_STRFTIME
 AC_FUNC_VPRINTF
 AC_FUNC_FORK
-- 
2.49.0

